src/DbUtils.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\StorageDriver\Teradata;
+
+use Doctrine\DBAL\Connection;
+use Keboola\TableBackendUtils\Escaping\Teradata\TeradataQuote;
+
+class DbUtils
```

Comment:
Neslo by nejak mocknut ten $connection a otestovat ze sa volaju query ake maju ?

Reply:
slo, pridano

Reply:
jsou to staticky metody, tak to je trochu hur citelny, ale myslim, ze snad OK